### PR TITLE
fixed dtype error, added test

### PR DESCRIPTION
**Pull request by jakubczakon**

### CODE REVIEW

File: tests/neptunelib/test_experiment.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+        leaderboard_entry = MagicMock()
+        dict_value = MagicMock()
+        dict_value.id = 0
+        leaderboard_entry.channels_dict_by_name = {'epoch_loss': dict_value}
+        leaderboard_entry.internal_id = 0
+
+        # then
+        experiment = Experiment(client, leaderboard_entry)
+        experiment.get_numeric_channels_values('epoch_loss')
```
[the remark this answers is not in the document]
Comment (jakubczakon) [reply tests/neptunelib/test_experiment.py]: I can do that but I was testing whether it created the dataframe not if it is a correct one.
So the test failed pre-fix but now it is working. 
Does that, in your opinion, change whether it should have an assert or not?

[the remark this answers is not in the document]
Comment (pitercl) [reply tests/neptunelib/test_experiment.py]: It doesn't. In a week noone will know that this was your motivation for building this test and any reader will wonder why we're not checking the result, which would provide additional functionality coverage and cost next to nothing.

[the remark this answers is not in the document]
Comment (jakubczakon) [reply tests/neptunelib/test_experiment.py]: Got it. I'll add one in a sec.

[the remark this answers is not in the document]
Comment (jakubczakon) [reply tests/neptunelib/test_experiment.py]: Done

**Pull request merged**